### DISCUSSION TITLE
Translations, i18n of errors

### DISCUSSION THREAD

**Discussion by zangab** [meteor/meteor#13318] [Feature Requests]
I installed `accounts-password@3.0.0` and an issue I have right now is the handling of error messages. As an example: https://github.com/meteor/meteor/blob/devel/packages/accounts-base/accounts_server.js#L1542

```
throw new Match.Error("User property must have exactly one field");
```

If my app is entirely in - let's say - german, I don't want any english error messages to be displayed. But as I've seen, I can't "translate" them or somehow override them.

Wouldn't it be possible to provide a functionality to override those existing error messages?
Let's say there's a json file with all your internal / existing error messages. However, I can put a file somehwere in my project that is merged on server startup so that my custom error messages (or translated ones) take presedence.

Or can this be achieved with a package like https://github.com/vazco/meteor-universe-i18n ??

**Comment by StorytellerCZ** [comment]
The fastest and dirtiest solution is to handle the error and have a function that will read what the error message is from the server and then return the translation to be displayed to the user.
Though I would appreciate some sort of official i18n package/approach that could be easily integrated into things like `react-intl`.
